### PR TITLE
Fortran Monthly Newsletter February 2022

### PR DESCRIPTION
**Pull request by jvdp1**
Here is a PR for the February 2022 newsletter:

* [x] PRs summary
* [x] GitHub stats data
* [x] Flang updates by @AlexisPerry 
* [x] LFortran updates by @certik

### CODE REVIEW

File: _posts/2022-02-01-Fortran-Newsletter-February-2022.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
+---
+layout: post
+title: "Fortran newsletter: January 2022"
```
Comment (jvdp1) [inline _posts/2022-02-01-Fortran-Newsletter-February-2022.md:3]:
```suggestion
title: "Fortran newsletter: February 2022"
```

File: _posts/2022-02-01-Fortran-Newsletter-February-2022.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+  [fpm.fortran-lang.org](https://fpm.fortran-lang.org/).
+  This website will provide user-oriented tutorials and how-to guides, as well
+  as developer-oriented reference documents and specifications.
+  We welcome all contributions to the fpm documentation, including translations
+  to other languages.
+  Please visit the [fpm-docs repo](https://github.com/fortran-lang/fpm-docs) to
+  get started.
+* We had our 22st Fortran Monthly call on January 17.
+  You can watch the recording below:
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/NSvL2yrefH8" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (jvdp1) [inline _posts/2022-02-01-Fortran-Newsletter-February-2022.md:163]: @milancurcic I couldn't find the recording on Youtube. Is it already there?

Comment (milancurcic) [reply _posts/2022-02-01-Fortran-Newsletter-February-2022.md:163]: Thanks for the reminder, I'll do it today and update the URL.

File: _posts/2022-02-01-Fortran-Newsletter-February-2022.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
```
Comment (milancurcic) [inline _posts/2022-02-01-Fortran-Newsletter-February-2022.md:163]: This month's video is processing and will be available shortly at this URL

```suggestion
  <iframe width="560" height="315" src="https://www.youtube.com/embed/93AObg7HsqM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

File: _posts/2022-02-01-Fortran-Newsletter-February-2022.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+
+Here's what's new in fpm:
+
+* [#630](https://github.com/fortran-lang/fpm/pull/630):
+  allow backfilling of current directory in fpm-new subcommand
+* [#646](https://github.com/fortran-lang/fpm/pull/646):
+  Respect user provided main-files
+* [#645](https://github.com/fortran-lang/fpm/pull/645):
+  Update module output directory command for flang-new/f18
+
```
Comment (awvwgk) [inline _posts/2022-02-01-Fortran-Newsletter-February-2022.md:97]:
```suggestion
* [fpm-docs#47](https://github.com/fortran-lang/fpm-docs/pull/47)
  [fpm-docs#46](https://github.com/fortran-lang/fpm-docs/pull/46)
  [fpm-docs#45](https://github.com/fortran-lang/fpm-docs/pull/45)
  [fpm-docs#44](https://github.com/fortran-lang/fpm-docs/pull/44)
  [fpm-docs#41](https://github.com/fortran-lang/fpm-docs/pull/41)
  [fpm-docs#39](https://github.com/fortran-lang/fpm-docs/pull/39):
  French translation
* [fpm-docs#43](https://github.com/fortran-lang/fpm-docs/pull/43):
  Add testing workflow for source examples
* [fpm-docs#40](https://github.com/fortran-lang/fpm-docs/pull/40):
  Update Spanish translation
* [fpm-docs#37](https://github.com/fortran-lang/fpm-docs/pull/37):
  zh_CN: Update Chinese translations

```

File: _posts/2022-02-01-Fortran-Newsletter-February-2022.md [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+  Add note about relocation of manifest reference
+* [#622](https://github.com/fortran-lang/fpm/pull/622) (WIP):
+  Cleanup the backend output
+* [#608](https://github.com/fortran-lang/fpm/pull/608) (WIP):
+  --env switch lets you specify the prefix of the compiler-related environment variables
+* [#539](https://github.com/fortran-lang/fpm/pull/539) (WIP):
+  Add parent packages into dependency tree
+* [#498](https://github.com/fortran-lang/fpm/pull/498) (WIP):
+  Compiler flags profiles
+
```
Comment (awvwgk) [inline _posts/2022-02-01-Fortran-Newsletter-February-2022.md:116]:
```suggestion
- [fpm-docs#42](https://github.com/fortran-lang/fpm-docs/issues/42) (WIP):
  Dutch translation

```

**Review by awvwgk — COMMENTED** [review]
There was a lot of activity at the fpm-docs regarding translations in French, Spanish, Chinese and Dutch this month as well.

**Comment by jvdp1** [comment]
> There was a lot of activity at the fpm-docs regarding translations in French, Spanish, Chinese and Dutch this month as well.

Should we keep `fpm-docs` in the `fpm` section? Or would it be better to create a new section for `fpm-docs`?

**Comment by awvwgk** [comment]
I would have the docs associated with fpm, which they are afterall.

**Comment by AlexisPerry** [comment]
Please see PR [372](https://github.com/fortran-lang/fortran-lang.org/pull/372) for Flang updates.

File: _posts/2022-02-01-Fortran-Newsletter-February-2022.md [hunk 6]
```diff
@@ -0,0 +1,222 @@
+---
+layout: post
+title: "Fortran newsletter: February 2022"
+category: newsletter
+authors: Jérémie Vandenplas, Alexis Perry-Holby
```
Comment (awvwgk) [inline _posts/2022-02-01-Fortran-Newsletter-February-2022.md:5]:
```suggestion
authors: Jérémie Vandenplas, Alexis Perry-Holby, Sebastian Ehlert
```

**Comment by certik** [comment]
@czgdp1807 is planning to add LFortran updates by tomorrow morning.

**Comment by czgdp1807** [comment]
N.B. - #373

**Review by awvwgk — APPROVED** [review]

File: _posts/2022-02-01-Fortran-Newsletter-February-2022.md [hunk 7]
```diff
@@ -0,0 +1,284 @@
...
+* Support substring references in NAMELIST input
+
+Call notes are recorded and available upon request [here](https://docs.google.com/document/d/10T-S2J3GrahpG4Ooif93NSTz2zBW0MQc_RlwHi0-afY). Please contact Alexis Perry-Holby at aperry@lanl.gov for document access.
+
+### LFortran
+
+**Compiling `stdlib` with `lfortran`**
+
+- [stdlib: Implement `AINT` intrinsic Procedure](https://gitlab.com/lfortran/lfortran/-/merge_requests/1638)
+- [Draft: [Sprint] Bug fixing to compile stdlib with LFortran](https://gitlab.com/lfortran/lfortran/-/merge_requests/1644)
```
Comment (czgdp1807) [inline _posts/2022-02-01-Fortran-Newsletter-February-2022.md:200]:
```suggestion
- [Draft: Sprint Bug fixing to compile stdlib with LFortran](https://gitlab.com/lfortran/lfortran/-/merge_requests/1644)
```

**Comment by jvdp1** [comment]
Thank you all for your contributions.
I will merge it tomorrow morning if there are no other comments. Feel free to merge it if you believe it can be merged as is.

**Comment by jvdp1** [comment]
> I think it is okay as is. We try to target the 1st of each month for publishing the newsletter. Let's merge it.

Good. I'll merge it.

**Pull request merged**